dockstore-webservice/src/main/java/io/dockstore/webservice/resources/ElasticSearchHealthCheck.java

```diff
@@ -0,0 +1,41 @@
+package io.dockstore.webservice.resources;
+
+import javax.ws.rs.core.Response;
+
+import com.codahale.metrics.health.HealthCheck;
+import io.dockstore.webservice.CustomWebApplicationException;
+import io.dockstore.webservice.resources.proposedGA4GH.ToolsExtendedApi;
+
+/**
+ * Checks that elastic search is healthy
+ */
+public class ElasticSearchHealthCheck extends HealthCheck {
+    private static final int LOWER_BOUND_SUCCESS_STATUS = 200;
+    private static final int UPPER_BOUND_SUCCESS_STATUS = 300;
+    private final ToolsExtendedApi toolsExtendedApi;
+
+    public ElasticSearchHealthCheck(ToolsExtendedApi toolsExtendedApi) {
+        this.toolsExtendedApi = toolsExtendedApi;
+    }
+
+    @Override
+    protected Result check() throws Exception {
+        // If elastic search is up with a valid index this should return healthy
+        Response response;
+        try {
+            response = toolsExtendedApi.toolsIndexSearch(null, null, null);
```

Comment:
Is this an expensive call, i.e., it places a heavy load on ES and/or returns a lot of data? This potentially get called a lot, if a monitor gets set up, so it should be lean.

Reply:
It could probably be simplified since it is just doing a search with no queries.

So this one is only called when you go to that healthcheck url. The other endpoint in metadata (metadata/elasticSearch) would be called by uptime robot.